### DISCUSSION TITLE
How to disable intel brew upgrade on ARM mac?

### DISCUSSION THREAD

**Discussion by leptonyu** [topgrade-rs/topgrade#910] [Q&A] [answered]
Intel brew is enabled on ARM.

**Comment by SteveLauC** [comment, marked as answer]
Looks like you cannot disable it unless you disable the whole brew step🙁

**Reply by leptonyu** [reply]
Thanks for the reply, I'll create a feature request.

**Reply by SteveLauC** [reply]
This is a known issue, see: https://github.com/topgrade-rs/topgrade/issues/515
